RadioItem/RadioItem.module.less

```diff
@@ -18,10 +14,22 @@
 			border-radius: @agate-radioitem-icon-border-radius;
 			color: transparent;
 			opacity: @agate-radioitem-icon-opacity;
+			margin: @agate-radioitem-icon-margin;
+		}
+
+		&.inline {
+			&.item {
+				padding: @agate-radioitem-inline-padding;
```

Comment:
I don't know why the padding should diverge depending on "inline". 
Did you add it because there is only an inline case in the GUI?
I think it's better to be unified.

Reply:
Okay. We can remove this

Reply:
OK, Let's unify.